src/createServer.js

```diff
@@ -1,11 +1,26 @@
 'use strict';
 
-// const express = require('express');
+const { userRouter } = require('./routes/users.router.js');
+const { expenseRouter } = require('./routes/expenses.route.js');
+const { resetUsers } = require('./services/users.service.js');
+const { resetExpenses } = require('./services/expenses.service.js');
+
+const express = require('express');
+const cors = require('cors');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  resetUsers();
+  resetExpenses();
```

Comment:
The `resetUsers` and `resetExpenses` functions are called during server creation. Ensure that these functions are intended to clear or reset data every time the server starts, as this might not be desirable in a production environment. Consider adding a condition or configuration to control when these resets should occur.